pkg/repository/repo.go

```diff
@@ -0,0 +1,64 @@
+package repository
```

Comment:
little doc snippet for what this is for?

Reply:
yup, will update.